web/styles/informational_overlays.css

```diff
@@ -126,3 +126,7 @@
 #search-operators-first-header {
     width: 40%;
 }
+
+.preserve-message-formatting-layout {
+    margin-bottom: unset !important;
```

Comment:
Is it possible to double-up on a class selector here rather than resorting to `!important`? Presumably `.rendered_markdown p` is what you're up against?

Reply:
Yes, you are right. The previous selector was overridden by the following selector from `rendered_markdown.css` file:
```
.rendered_markdown p {
    margin: 0 0 var(--markdown-interelement-space-px);
}
```

Based on the above review, I think we can craft the following selector, taking in mind that the changes we want to apply are isolated in the Message Formatting section:
```
#message-formatting {
    & .preserve-message-formatting-layout {
        margin-bottom: unset;
    }
}
```
Thank you. Let me know if any further adjustments are needed.